homeassistant/components/velbus/light.py

```diff
@@ -32,22 +45,22 @@ class VelbusLight(VelbusEntity, LightEntity):
 
     _attr_supported_feature = SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION
 
-    def __init__(self, channel):
+    def __init__(self, channel: VelbusChannel) -> None:
         """Initialize the dimmer."""
         super().__init__(channel)
         self._attr_name = self._channel.get_name()
 
     @property
-    def is_on(self):
+    def is_on(self) -> Any:
```

Comment:
Please avoid using Any as much as possible. The interface of the different entity classes that we implement has specific type requirements. We are not allowed to deviate outside that.